src/jdk.jpackage/macosx/classes/jdk/jpackage/internal/model/MacDmgPackageMixin.java

```diff
@@ -32,6 +32,14 @@ public interface MacDmgPackageMixin {
 
     Optional<Path> icon();
 
+    /**
+     * Will be set to root of created install dir for deletion once it is no
+     * longer needed.
+     *
+     * @return Root of created install dir
+     */
+    Optional<Path> installDirDeleteRoot();
```

Comment:
Why is this information a part of the model? Can't dmg packager figure it out on its own?